dev/udf_proxy.h

```diff
@@ -13,9 +13,9 @@
 
 namespace sqlite_orm {
     namespace internal {
-        /*
-         *  Returns properly allocated memory space for the specified application-defined function object
-         *  paired with an accompanying deallocation function.
+        /*
+         *  Returns properly allocated memory space for the specified application-defined function object
+         *  paired with an accompanying deallocation function.
```

Comment:
I'd like to know what changed here

Reply:
New-line endings. You might want to enable the option to hide whitespace when comparing the diff.